master/static/srv/notebook-entrypoint.sh

```diff
@@ -18,6 +18,31 @@ set -x
 test -f "${STARTUP_HOOK}" && source "${STARTUP_HOOK}"
 set +x
 
+if [ "$PACHD_ADDRESS" != "" ] && [ "$SKIP_PACHYDERM_INSTALL" != "true" ]; then
+    proxy_dns=$(echo $PACHD_ADDRESS | sed 's/grpc:\/\/pachd/pachyderm-proxy/' | sed 's/:30650//') 
```

Comment:
I can't review the accuracy of this. Maybe someone from the pachyderm side should.

BTW, so far, as far as I know, the way the pachyderm integration is written in Determined right now it doesn't know/care that is it being installed by the unified installer. So what I'm getting at some is I'd try to not tie the correctness of this code to that condition.

Reply:
I think in general, we haven't tested what happens when we're not installed using Pachyderm + Determined in the same Kubernetes cluster. 

It'll take me a while to get this set up and tested, but I'll commit to doing that.

The way I understand the routing, we would never talk to pachd directly if we're separate clusters, and all routing would go through proxy. I'll have to understand what that looks like, and if that API call is available by just curling `integrations.pachyderm.address`

Reply:
For this PR I want to limit the scope to only the combined install. If we want to support having them in separate clusters I would like that to be a follow on PR that the Det Eng team handles. This PR is to unblock current unified install users.